### PR TITLE
Depth Texture clearing fix for Overlay Camera

### PR DESCRIPTION
**Pull request by jonuuukas**
---
### Purpose of this PR
---
Removing SetGlobalTexture to black/white for Overlay cameras, as it's main intention is to reclear it every frame and it's guaranteed Base camera will render first and do this. For Overlay cameras - there is specific checkbox that forces clear of the depth.

This also changes behaviour for failing tests, as it seems like depth is recreated from scratch and might create ordering problems (needs more investigation, but it's hard due to no repro on local devices) so disabling Depth Texture for the overlay cameras in URP_PostPro tests 145 as they only need previous camera's depth and no need to recreate it

### Testing status
---
URP_PostPro is green, tested manually, running rest of Universal_Split

### CODE REVIEW

File: com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs [hunk 1]
```diff
@@ -722,7 +722,7 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
             }
 
             // Set the depth texture to the far Z if we do not have a depth prepass or copy depth
-            if (!requiresDepthPrepass && !requiresDepthCopyPass)
+            if (cameraData.renderType == CameraRenderType.Base && !requiresDepthPrepass && !requiresDepthCopyPass)
```
Comment (martint-unity) [inline com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs:725]: Could you add to the comment above the reasoning behind this?

**Review by martint-unity — CHANGES_REQUESTED** [review]

File: com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs (hunk 1 above)
Comment (jonuuukas) [reply com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs:726]: added comment in e6049c62a06b9f1374cc18d24c8bf3e1a4bb16a2

**Review by martint-unity — APPROVED** [review]
LGTM

**Review by ernestasKupciunas — APPROVED** [review]
Approving without testing. The developer has successfully tested his changes.

**Pull request merged**